rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/testing_utils.kt

```diff
@@ -680,6 +680,15 @@ fun compileAllMutes(
                         ?: error("resource $dbFile.json not found in $metadataPaths")
                 }
             )
+
+            dspfConfig = DspfConfig { displayFile ->
```

Comment:
I have no information about which `displayFile` is not found. See the implementation above as an example of the correct handling.